org.electronjs.Electron2.BaseApp.yml

```diff
@@ -94,3 +94,46 @@ modules:
         x-checker-data:
           type: git
           tag-pattern: ^v([\d.]+)$
+
+  # Chromium dynamically loads the Speech Dispatcher library libspeechd.so.2.
+  # If present, and --enable-speech-dispatcher flag is passed, Web Speech API will be enabled.
+  # This section is borrowed and tweaked from:
+  # https://github.com/flathub/net.lutris.Lutris/blob/76e94a0b80ef3ebc1b9a6b61f47d736f5ddd772c/net.lutris.Lutris.yml#L495
+  # https://gitlab.archlinux.org/archlinux/packaging/packages/speech-dispatcher/-/blob/414baaf78b5fe416df88a89ac18d2dd579a0c653/PKGBUILD
+  - name: libspeechd
+    config-opts:
+      - --disable-static
+      - --with-ibmtts=no
+      - --with-kali=no
+      - --with-baratinoo=no
+      - --with-voxin=no
+      - --without-flite
+      - --disable-python
+      - --with-systemdsystemunitdir=/app/lib/systemd/system/speech-dispatcherd.service
+    no-make-install: true
+    post-install:
+      - cd ./src/api/c && make install
+    cleanup:
+      - "*.la"
+      - "*.a"
+      - /include
```

Comment:
Does electron need the base library libspeechd.so only If so further things should be cleaned up https://github.com/flathub/org.mozilla.firefox.BaseApp/blob/7c7fe8222b3066786868174f9462409122ac123b/org.mozilla.firefox.BaseApp.json#L139-L146

Reply:
> [Chromium source code for proof](https://source.chromium.org/chromium/chromium/src/+/refs/tags/122.0.6261.111:content/browser/speech/tts_linux.cc;l=251).

Seems like so, please cleanup rest of the stuff

Reply:
Not needed.